### DISCUSSION TITLE
Password management

### DISCUSSION THREAD

**Discussion by PsySuck** [neos-sdi/adfsmfa#258] [Q&A] [answered]
Hi.
I can't find password management. Where i can find it?
I enabled it mmc. Disable adfs endpoint.
Currently i am using 3.1.2207.0

**Comment by redhook62** [comment, marked as answer]
Sorry, for me all this is displayed correctly.
It's the kind of functionality that hasn't changed for several years.

**Comment by PsySuck** [comment]
My question where i can find the button?
Here is setting and my user configuration
<img width="787" alt="Screenshot 2022-10-14 at 16 13 37" src="https://user-images.githubusercontent.com/87027160/195869507-e0bffd77-c304-4267-8427-f470c5a736eb.png">
<img width="939" alt="Screenshot 2022-10-14 at 16 14 05" src="https://user-images.githubusercontent.com/87027160/195869513-c1eedfee-2c3f-4064-8d0c-8a2f46c3396b.png">
<img width="410" alt="Screenshot 2022-10-14 at 16 17 12" src="https://user-images.githubusercontent.com/87027160/195869516-0593593c-143c-4975-b2aa-23f48db3346c.png">

**Comment by redhook62** [comment]
Your user certainly does not have the right to change his password. Check in ADDS

**Comment by PsySuck** [comment]
Do you mean service user or my user?
For my user - Adfs change password/Ctrl-alt-delete works.

**Comment by redhook62** [comment]
Activate password management in the "security" tab in the console

**Comment by PsySuck** [comment]
Thank you.
I am able to reset password.
